UnofficialSteamAuthenticator/App.xaml.cs

```diff
@@ -14,9 +14,9 @@ namespace UnofficialSteamAuthenticator
     /// <summary>
     /// Provides application-specific behavior to supplement the default Application class.
     /// </summary>
-    public sealed partial class App : Application
```

Comment:
I don't think relying on it because it was templated in VS is... wise. There are a few things templated to make it easier for new users to understand, but cumbersome for well-known users.

You want me to add it again?